instrumentation/apache-camel-2.20/javaagent-unit-tests/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/SanitizationTest.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel.decorators
+
+import org.apache.camel.Exchange
+import org.apache.camel.Message
+import spock.lang.Specification
+
+class SanitizationTest extends Specification {
```

Comment:
I've just noticed it now, but if you're using parameterized test case names you should add `@Unroll` on the class/test methods

Reply:
Thanks, I'll add it right away 👍